src/main/java/io/strimzi/kafka/bridge/mqtt/utils/MappingRule.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.bridge.mqtt.utils;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Represents a Mapping Rule in the Topic Mapping Rules(TOMAR). Mapping rules are used to define how MQTT topics should be mapped to Kafka topics.
+ * E.g.: a valid mapping rule would look like this in the TOMAR file:
+ * {
+ * "kafkaTopic": "sensors_{sensorId}_data",
+ * "mqttTopic": "sensors/{sensorId}/data"
+ * }
+ * and like this in the MappingRule class:
+ * MappingRule(kafkaTopicTemplate=sensors_{sensorId}_data mqttTopicPattern= sensors/{sensorId}/data)
+ */
+public class MappingRule {
+    @JsonProperty("kafkaTopic")
+    private String kafkaTopicTemplate;
```

Comment:
Would it make more sense to call this `kafkaTopicPattern` instead of `kafkaTopicTemplate` for symmetry with `mqttTopicPattern`?

Reply:
+1

Reply:
Yeah I was thinking the same while reading the code again. I agree with Antonio.